docs/_manage/making-changes.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Making Integration Changes in Platform UI and CLI
+order: 6
+layout: post-toc
+---
+
+# Making Integration Changes in Platform UI and CLI
+
+Before making updates to your integration, it's important to consider the potential impact on user migration and existing Zaps. Ensuring your API and Zapier integration remains backwards compatible is crucial to avoid disruption to users. However, we acknowledge certain changes are sometimes necessary and avoidable. In such cases, consider the best practices for implementation.
+
+## Affects of Different Changes (Versioning Matrix)
+
+The matrix below illustrates the impact of different changes on promotions and migrations for public integration. Refer to our best practices to facilitate the upgrade process for you and your users.
```

Comment:
Nitpick: Unsure about this grammar, but I believe its best practice. Feel free to ignore!
```suggestion
The matrix below illustrates the impact of different changes on promotions and migrations for public integration. Refer to our best practices to facilitate the upgrade process for yourself and your users.
```